packages/web/components/templates/homeFeed/HomeFeedContainer.tsx

```diff
@@ -975,7 +989,9 @@ function HomeFeedGrid(props: HomeFeedContentProps): JSX.Element {
       )}
       {showRemoveLinkConfirmation && (
         <ConfirmationModal
-          message={'Are you sure you want to remove this link?'}
+          message={
+            'Are you sure you want to remove this link? All your highlights will be deleted.'
```

Comment:
@jacksonh I mention the highlights deleted in the confirmation modal. The wording could be changed

Reply:
How about:

```
Are you sure you want to remove this item? All associated notes and highlights will be deleted.'
```

Reply:
Better!

Reply:
I'll make it the same on iOS

Reply:
Okay. Updated